### PR TITLE
Add CosmoPower

### PR DESCRIPTION
**Pull request by joezuntz**
Created using `grayskull pypi cosmopower`

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by joezuntz** [comment]
@conda-forge/help-python - this is ready for review

File: recipes/cosmopower/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  commands:
+    - pip check
+  requires:
+    - python >={{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/alessiospuriomancini/cosmopower
+  summary: Machine Learning - accelerated Bayesian inference
+  license: GPL-2.0-or-later
```
Comment (xhochy) [inline recipes/cosmopower/meta.yaml:43]: Upstream specifies GPLv3: https://github.com/alessiospuriomancini/cosmopower/blob/cb323858f5567b61d3de71aadb452c95ef908984/setup.py#L38

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/cosmopower/meta.yaml (hunk 1 above)
Comment (joezuntz) [reply recipes/cosmopower/meta.yaml]: When I had GPL-3.0 (as generated by Grayskull) your linter complained.

The list on this page didn't include GPL-3: https://conda-forge.org/docs/maintainer/adding_pkgs/#spdx but I see on the page it links to that this needs to be GPL-3.0-only or GPL-3.0-or-later based. I will change it now.

**Comment by joezuntz** [comment]
@xhochy - many thanks - I have made the change.

File: recipes/cosmopower/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% set name = "cosmopower" %}
+{% set version = "0.2.0" %}
+{% set python_min = 3.9 %}
```
Comment (xhochy) [inline recipes/cosmopower/meta.yaml:3]: This should no longer be required.
```suggestion
```

**Review by xhochy — APPROVED** [review]

**Pull request merged**